index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome-Books-ES6</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <div class="navigation-title">Awesome Books</div>
+        <div class="">
+            <ul class="navigation-list">
+                <li id="list" class=" dark-red navigationlist-item"><a>List</a></li>
+                <li id="add" class=" navigationlist-item"><a>Add new</a></li>
+                <li id="contact" class="navigationlist-item"><a>Contact</a></li>
+            </ul>
+        </div>
+    </nav> 
+    <main>
+    <section class="date-section">
+        <p class="date">July 5th, 16:17:01 pm</p>
+    </section>
```

Comment:
- [ ] Kindly make sure to remove this static `time and date` and replace it with the one accessed in your code using the `luxon` library.

Reply:
Ok, Omitted that line of code, thanks for your suggestion